morphia/src/main/java/org/mongodb/morphia/DatastoreImpl.java

```diff
@@ -250,6 +260,40 @@ public void ensureCaps() {
     }
 
     @Override
+    public void enableDocumentValidation() {
+        for (final MappedClass mc : mapper.getMappedClasses()) {
+            Validation validation = (Validation) mc.getAnnotation(Validation.class);
+            if (validation != null) {
+                String collectionName = mc.getCollectionName();
+                CommandResult result = getDB()
+                    .command(new BasicDBObject("collMod", collectionName)
+                                 .append("validator", Document.parse(validation.value()))
```

Comment:
Nit.

Seems weird to mix DBObject and Document - why not use `BasicDBObject.parse` ?